extension/src/commands/command-controller.ts

```
@@ -15,16 +16,20 @@ export class CommandController {
 
   // Registers a command with VS Code so it can be invoked by the user.
   #registerCommand (command: string, callback: (...args: any[]) => any): void {
-    // DEBUG_LOG('Start Registering command ' + command)
+    try {
+      // DEBUG_LOG('Start Registering command ' + command)
 
-    const cmd: Disposable = commands.registerCommand(command, callback)
+      const cmd: Disposable = commands.registerCommand(command, callback)
 
-    // ext.ctx.subscriptions.push(commands.registerCommand(command, callback))
+      // ext.ctx.subscriptions.push(commands.registerCommand(command, callback))
 
-    this.cmds.push(cmd)
+      this.cmds.push(cmd)
 
-    // ext.ctx.subscriptions.push(cmd)
-    // DEBUG_LOG('Registered command ' + command)
+      // ext.ctx.subscriptions.push(cmd)
+      // DEBUG_LOG('Registered command ' + command)
+    } catch (err) {
+      Telemetry.captureException(err)
+    }
```

This silences the error for the user, which could be confusing. Maybe `throw` here?